templates/install/install.tpl

```diff
@@ -22,8 +22,8 @@
 	<br />
 
 	<div class="app__contentPanel">
-		{fbvFormSection label="installer.locale" for="locale" style="position: absolute;"}
-			{fbvElement type="select" name="locale" id="installLanguage" from=$localeOptions selected=$locale translate=false size=$fbvStyles.size.SMALL subLabelTranslate=true}
+		{fbvFormSection label="common.language" for="installLanguage" style="position: absolute;"}
```

Comment:
I've added this field to change the language during the installation, but it needs some styling, I can do it, but I decided to ensure it's not going to be removed :)

Another generic improvement would be to choose a default locale considering the locales from the `Accept-Language` header.

Reply:
:+1: agreed, an `Accept-Language` check would be great -- this should be done in the session manager where sessions are initiated.

Reply:
(But no need to tackle that here -- in fact better file a new issue for it.)